### ISSUE TITLE
update to latest stable Postgresql available on aws

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#1052]
Find out what the latest version of Postgresql is that's available for the non-experimental RDS and switch to it.

Turns out, I ran into a diesel regression that bites us when using 12.8 and 12.12 but doesn't bite us using 13.8. I'm going to report that regression to diesel because it might bite others, but it's been a long while since we've updated Postgres and may as well do it "now".

- [x] Find out what's the latest available to us (14.4)
- [x] Spin up a new instance
- [x] Verify that I can connect via `psql`
- [x] `time pg_dump -Z9 -Fc -h mb2-prod.cf9j64crpawc.us-east-2.rds.amazonaws.com -U mb2 | pg_restore -h mb2-14.cf9j64crpawc.us-east-2.rds.amazonaws.com -U mb2 -Cc -d template1`
- [x] Alter appropriate environment variables
- [x] Restart mb2
- [x] Delete the old instance

**Comment by ctm** [comment]
I'm dumping the database. It's big enough that it's been going on a while.

**Comment by ctm** [comment]
The load appears to be running fine. I didn't time the dump, but the new instance is less than two hours old and I think I created it before I started the dump (which was silly; I should have started the dump as soon as I knew I wanted to upgrade). So, I think I'll have the db loaded well before the start of this evening's tournament, but there may be trouble with people's sessions when we jump back in time.

**Comment by ctm** [comment]
Nope. The load took too long. I should have done it as a pipe, so I saved the `.psql` file on my QNAP and then started a pipeline, just to see if there were any errors, then I killed it so that the dump itself won't interfere with this evening's tournament.

Since I'm not saving the file when I'm using a pipe, I should just run the commands on `devctm.com` and use the internal ip address. That way I don't have to worry about the slow bandwidth between AWS and my home. I'll experiment with that after this evening's game is working. Then I'll fire up the real transfer in the middle of the night.
